src/JiraCli/Extensions/JiraExtensions.cs

```diff
@@ -47,8 +48,17 @@ public static JToken ExecuteRequestRaw(this IJiraRestClient jiraRestClient, Meth
                 Value = jsonRequestBody
             });
 
-            var response = jiraRestClient.ExecuteRequest(restRequest);
-            return response.StatusCode != HttpStatusCode.NoContent ? JToken.Parse(response.Content) : new JObject();
+            try
+            {
+                var response = jiraRestClient.ExecuteRequest(restRequest);
+                return response.StatusCode != HttpStatusCode.NoContent ? JToken.Parse(response.Content) : new JObject();
+            }
+            catch (System.Exception ex)
+            {
+                Debug.Write(ex);
```

Comment:
Please use Log.Error(ex) instead.

At the top of the class, add this:

private static readonly ILog Log = LogManager.GetCurrentClassLogger();